app/uk/gov/hmrc/hmrcemailrenderer/templates/voac/voaConfirmationMessageAlert.scala.txt

```diff
@@ -0,0 +1,19 @@
+@(params: Map[String, Any])Valuation Office Agency Contact Form
+
+Enquiry summary
+
+Property address: @params("propertyAddress")
+
+Enquiry type: @params("enquiryCategoryMsg")
+              @params("subEnquiryCategoryMsg")
+
+Personal details: @params("firstName")
+                  @params("lastName")
+                  @params("email")
+                  @params("contactNumber")
+
+Message: @params("message")
+
+From Valuation Office Agency Contact Form
```

Comment:
Can you make this say 
"VOA Internal use only From Valuation Office Agency Contact Form"